src/main/java/string/Exer11.java

```diff
@@ -0,0 +1,40 @@
+package string;
+
+public class Exer11 {
+
+    public String subNumbers(String num1, String num2) {
+        StringBuilder result = new StringBuilder();
+        if ((num2.length() > num1.length()) || (num2.length() == num1.length() && num2.compareTo(num1) > 0)) {
+            return "-" + subNumbers(num2, num1);
+        }
+        int temp;
+        int carry = 0;
+
+        for (int i = 0; i < num1.length(); i++) {
+            temp = digitFromRight(num1, i) - digitFromRight(num2, i) + carry;
+            if (temp < 0) {
+                temp = temp + 10;
+                carry = -1;
+            } else {
+                carry = 0;
+            }
+            result.append(temp);
```

Comment:
Để khỏi reverse kết quả sau đó

```suggestion
            result.insert(0, temp);
```

Reply:
fixed